test/sandbox/app/components/nested_component.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+class NestedComponent < ViewComponent::Base
```

Comment:
Can we make this component a bit simpler, while still having the test fail? I think it would help understanding what's going on.